src/Robo/Plugin/Commands/DevelopmentModeCommands.php

```diff
@@ -337,12 +339,13 @@ protected function landoUri($siteDir): string
     /**
      * Refresh database on Tugboat.
      *
-     * @return null|\Robo\Result
+     * @return \Robo\Result
      *   The task result.
      */
     public function databaseRefreshTugboat(): Result
     {
         $this->io()->title('refresh tugboat databases.');
+        $result = null;
```

Comment:
`null` is ok. It would be better if Robot provided a way of instantiating an empty \Robo\Result, so the signature could always return a Robo Result instead of `null`.

Reply:
I like this idea. ☝🏼 

If this idea doesn't pan out, does this mean that our docblock should be `@return \Robo\Result|null`?

Reply:
I like it too. PRs welcome! 😉